tests/testthat/test-estimate_cox.R

```diff
@@ -0,0 +1,358 @@
+#' @title Specifications test-estimate_cox.R
+#' @section Last updated by: Steven Haesendonckx/Alexandra Papadopoulou
+#' @section Last update date: 2022-05-02
+#'
+#' @section List of tested specifications
+#' T1.  The function accepts a `data.frame` `tibble` or `data.table`
+#' T1.1 No error when `data` is of class `data.frame`
+#' T1.2 No error when `data` is of class `tibble`
+#' T1.3 No error when `data` is of class `data.table`
+#' T1.4 An error when `data` is of an unexpected class, eg `list`
+#' T1.5 An error when `data` is NULL
+#' T2. The function relies on the presence of two numeric variables, specified through `AVAL` and `CNSR`, to be present in `data`
+#' T2.1 An error when colname specified through `AVAL` is not present in `data`
+#' T2.2 An error when colname specified through `AVAL` is not numeric
+#' T2.3 No error when the colname specified through `AVAL` is not the proposed default
+#' T2.4 An error when the colname specified through `CNSR` is not present in `data`
+#' T2.5 An error when the colname specified through `CNSR` is not numeric
+#' T2.6 No error when the colname specified through `CNSR` is not the proposed default
+#' T3.  The user can specify the equation
+#' T3.1 An error when the columns, specifying the equation are not available in `data`
+#' T3.2 No error when equation is NULL
+#' T3.3 When no equation is specified, the 'null model' is displayed
+#' T3.4 When the equation has a term with only one factor, expect an error
+#' T4.  The function removes all rows with NA values inside CNSR or AVAL
```

Comment:
T4 should test conf.int see estimate_cuminc and below should be upgraded by 1 eg T4 becomes T5

Reply:
Done